dev-python/construct/construct-2.5.2.ebuild

```diff
@@ -7,19 +7,14 @@ PYTHON_COMPAT=( python{2_7,3_4} pypy )
 
 inherit distutils-r1
 
-MY_PN="construct"
-MY_P="${MY_PN}-${PV}"
-
 DESCRIPTION="A powerful declarative parser for binary data"
 HOMEPAGE="http://construct.wikispaces.com/ https://pypi.python.org/pypi/construct"
-SRC_URI="mirror://pypi/${MY_PN:0:1}/${MY_PN}/${MY_P}.tar.gz"
+SRC_URI="mirror://pypi/${PN}/${PN}/${P}.tar.gz"
```

Comment:
This is wrong. It should be `${PN::1}` for the first bit (just the first letter).